### PR TITLE
Flagsmith integration

### PR DESCRIPTION
**Pull request by dabeeeenster**
### What does this PR do?

Integrates Datadog with Flagsmith

### Motivation

We have had several customers request this feature

### Review checklist

- [x] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [x] Feature or bugfix has tests
- [x] Git history is clean
- [x] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: flagsmith/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+Flagsmith does not include any service checks.
+
+### Events
+
+Flagsmith does not include any events.
+
+## Troubleshooting
+
+Need help? Check out our [documentation](https://docs.flagsmith.com/integrations/datadog/) or [contact us][https://www.flagsmith.com/].
```
Comment (cswatt) [inline flagsmith/README.md:36]:
```suggestion
Need help? Check out the [Flagsmith documentation](https://docs.flagsmith.com/integrations/datadog/) or [contact Datadog Support][https://docs.datadoghq.com/help/].
```

**Review by cswatt — CHANGES_REQUESTED** [review]
You'll also need an empty `metadata.csv`.

**Comment by dabeeeenster** [comment]
Hi @cswatt  - I've applied all those changes. Thanks for looking into the PR so quickly.

**Comment by hithwen** [comment]
Hallo, you need to ensure automated validation passes. At the moment is failig because you need to include an entry in `.github/CODEOWNERS` file in this same PR

**Comment by dabeeeenster** [comment]
OK thanks that's done.

**Comment by hithwen** [comment]
> OK thanks that's done.

There are other failures, you should be able to see the build results. Let me know if you have specific questions.

**Comment by dabeeeenster** [comment]
Hi - Tests are passing now - thanks!

**Review by cswatt — APPROVED** [review]
This looks good from a documentation perspective!

**Comment by dabeeeenster** [comment]
Hi - is there anything else you need from me for this to be merged? Thanks!

**Review by hithwen — APPROVED** [review]

**Pull request merged**